ONSdigital/ras-frontstage
# RAS-739: Add modal to base template

# What and why?
There is a requirement that people logged into frontstage need to be notified that they will be logged out 60 seconds before their session times out. They can either stay on the page and their timeout counter will be refreshed or they will be redirected to the login screen.

These code changes will achieve this.

# How to test?
1. Sign into frontstage with a user account
2. Once redirected to the todo survey list keep the page loaded and do not navigate anywhere whilst logged in with the above user account
3. After 59 minutes have elapsed, a message will appear on the screen with a counter
4. Test both the following:
- Click the 'Continue survey' button and you should be on the same screen that you you originally opened
-  Click the 'Continue survey' button once the timer has counted down. This will redirect you to the login page

# Jira
https://jira.ons.gov.uk/browse/RAS-739
